src/System.Private.CoreLib/shared/System/Environment.Unix.cs

```diff
@@ -75,14 +75,16 @@ private static string GetFolderPathCore(SpecialFolder folder, SpecialFolderOptio
             {
                 Debug.Assert(option == SpecialFolderOption.Create);
 
+#pragma warning disable CS8634 // TODO-NULLABLE: Remove warning disable when nullable attributes are respected
```

Comment:
Why did you start getting a warning here?

Reply:
Because I changed how EnsureInitialized is annotated.